helper/resource/aliases.go

```diff
@@ -0,0 +1,109 @@
+package resource
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
```

Comment:
Do we need to add the exports from the new `id` package to this set of aliases as well?

![image](https://user-images.githubusercontent.com/8650838/224307982-6dfbc303-125a-4c63-adc8-84ff7c706b9c.png)

Reply:
+1

Reply:
++1